src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
+// Змінні
+$color-grey: #eee;
+$color-black: #000;
+$color-pink: #ffbfcb;
+$border-width: 1px;
+$day-size: 100px;
+$font-size: 30px;
+$gap: 1px;
+$padding: 10px;
+$hover-move: 20px;
+$animation-duration: 0.5s;
+
+// Основні стилі
```

Comment:
```suggestion
```

Comments are unnecessary, there is a practice of leaving comments in the code, but this happens very rarely and only in places with critically complex logic.

Nowadays, comments in code raise more questions and suspicion of use AI ))